models/ipns.go

```diff
@@ -85,7 +100,8 @@ func (im *IpnsManager) UpdateIPNSEntry(ipnsHash, ipfsHash, key, networkName stri
 	return &entry, nil
 }
 
-func (im *IpnsManager) CreateEntry(ipnsHash, ipfsHash, key, networkName string, lifetime, ttl time.Duration) (*IPNS, error) {
+// CreateEntry is used to create a brand new IPNS entry in our database
+func (im *IpnsManager) CreateEntry(ipnsHash, ipfsHash, key, networkName, username string, lifetime, ttl time.Duration) (*IPNS, error) {
```

Comment:
not a big deal, but we should be careful of some function signatures getting too large - use of option structs are a good way to manage the complexity of function interfaces

Reply:
:thinking: good idea, that would be especially useful for IPNS record management!